docs/api/inputs/chat.md

```diff
@@ -257,4 +339,3 @@ configuration with a dictionary conforming to the config.
 
 ```{eval-rst}
 .. autoclass:: marimo.ai.ChatAttachment
-```
```

Comment:
This deletion doesn't seem to be breaking `ChatAttachment` class from rendering in the docs deployed; hope this is alright?